samples/ORKCatalog/ORKCatalog/TaskListRow.swift

```diff
@@ -764,13 +764,13 @@ enum TaskListRow: Int, Printable {
         var consentSections: [ORKConsentSection] = consentSectionTypes.map { contentSectionType in
             let consentSection = ORKConsentSection(type: contentSectionType)
             
-            consentSection.summary = self.loremIpsumShortText
+            consentSection.summary = self.loremIpsumLongText
             
             if contentSectionType == .Overview {
                 consentSection.htmlContent = htmlContentString
             }
             else {
-                consentSection.content = self.loremIpsumLongText
+                consentSection.content = self.loremIpsumVeryLongText
```

Comment:
I'd prefer you did this testing in ORKTest rather than ORKCatalog. ORKCatalog is meant to be representative of how it should actually appear, rather than show worst case.